src/utils/validate.ts

```diff
@@ -15,8 +17,14 @@ export const validateCryptoAddress =
   (coin: string) =>
   async (address: string): Promise<string | boolean> => {
```

Comment:
why was this async? promises in validation are usually not great

Reply:
only noticed because you switched the references to this func to use `await`

Reply:
The validateCrypto function is async, I checked inside and it doesn't have an async function inside so I turned it into a sync func.